apps/rule-manager/client/src/ts/components/RulesTable.tsx

```diff
@@ -122,16 +126,25 @@ const RulesTable = () => {
 
             }
         </EuiFlexGrid>
-        {rules &&
-            <EuiInMemoryTable
-                tableCaption="Demo of EuiInMemoryTable"
-                items={rules}
-                columns={columns}
-                pagination={true}
-                sorting={sorting}
-                search={search}
-            />
-        }
+
+        <EuiFlexGroup>
+            <EuiFlexItem grow={2}>
+                {rules &&
+                    <EuiInMemoryTable
+                        tableCaption="Demo of EuiInMemoryTable"
+                        items={rules}
+                        columns={columns}
+                        pagination={true}
+                        sorting={sorting}
+                        search={search}
+                    />
+                }
+            </EuiFlexItem>
+            <EuiFlexItem grow={1}>
+                <RuleForm fetchRules={fetchRules}/>
```

Comment:
Non-blocking – wonder if this is better expressed as `onCreate` or `onChange`, that is to say, our RuleForm shouldn't know about fetching rules, but should provide a hook to do something when users click 'create' (or, later on, 'update')

Reply:
I've gone with `onRuleUpdate` for now as that's specifically when it's called by the RuleForm currently